analyzer-java/src/main/java/com/infosupport/ldoc/analyzerj/AnalysisVisitor.java

```diff
@@ -78,13 +79,27 @@ public AnalysisVisitor(SymbolResolver resolver) {
     this.resolver = resolver;
   }
 
-  /** Resolves the given type to a String with its fully-qualified class name. */
-  private String resolve(Type type) {
+  /** Resolves the given node to a String with its fully-qualified class name.
```

Comment:
Nit: For multi-line Javadoc comments, usually the `/**` gets its own line. The `* */` is not written with a space.